dotnet-datetime/StopWatchCSharp/StopWatchCSharpTests/StopWatchUnitTests.cs

```diff
@@ -0,0 +1,72 @@
+using System.Diagnostics;
+
+namespace StopWatchCSharpTests
+{
+    [TestClass]
+    public class StopWatchUnitTests
+    {
+        private readonly int[] _numArray;
+        private readonly StopWatchMethods _stopWatchMethods = new();
+
+        public StopWatchUnitTests() 
+        {
+            _numArray = _stopWatchMethods.CreateRandomArray(20000);
+        }
+
+        [TestMethod]
+        public void GivenBubbleSortAlgorithm_WhenInvoked_VerifyStopwatchSuccess()
+        {
+            var stopWatch = _stopWatchMethods.BubbleSort(_numArray, _numArray.Length);
+
+            Assert.IsInstanceOfType(stopWatch, typeof(Stopwatch));
+            Assert.IsInstanceOfType(stopWatch.Elapsed, typeof(TimeSpan));
+            Assert.IsFalse(stopWatch.IsRunning);
```

Comment:
The only meaningful assert here is the IsFalse(stopWatch.IsRunning). We could add an additional assert to ensure the stopwatch elapsed time is not zero after calling the sort method. That would be meaningful in the context of the test.